pycbc/events/threshold_cpu.py

```diff
@@ -32,7 +32,7 @@ def threshold_numpy(series, value):
     arr = series.data
     locs = numpy.where(arr.real**2 + arr.imag**2 > value**2)[0]
     vals = arr[locs]
-    return locs, vals
+    return vals, locs
```

Comment:
I think we need to be consistent on the output order. The other single function thresholding functions have the original order. The exception is the threshold and cluster function, which I agree is inconsistent overall, but I think this change makes it less so. Can you get away with just changing the order where you are calling this function?

Reply:
@ahnitz You mean swap order here https://github.com/idorrington92/pycbc/blob/26c68e2d0781dd76c7b3a73af4cbf034ccd1dc42/pycbc/filter/matchedfilter.py#L339 instead?

Reply:
Yes, that's what I mean.